From 6c46cf889cfd831ecce288005f85c7a254a64cb1 Mon Sep 17 00:00:00 2001
From: Jonathan Bell <jonathan@raspberrypi.org>
Date: Thu, 18 Jul 2019 13:05:35 +0100
Subject: [PATCH 712/725] bcm2711_defconfig: enable PCI portbus support (and
 implicitly, PCIe AER)

PCIe advanced error reporting is supported by the root complex, so make
use of it.

Signed-off-by: Jonathan Bell <jonathan@raspberrypi.org>
---
 arch/arm/configs/bcm2711_defconfig | 2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm/configs/bcm2711_defconfig
+++ b/arch/arm/configs/bcm2711_defconfig
@@ -33,6 +33,8 @@ CONFIG_ARCH_BCM2835=y
 CONFIG_ARM_LPAE=y
 # CONFIG_CACHE_L2X0 is not set
 CONFIG_PCI=y
+CONFIG_PCIEPORTBUS=y
+# CONFIG_PCIEASPM is not set
 CONFIG_PCI_MSI=y
 CONFIG_PCIE_BRCMSTB=y
 CONFIG_SMP=y
